lib/OnyxConnectionManager.ts

```diff
@@ -42,6 +42,16 @@ type ConnectionMetadata = {
      * The last callback key returned by `OnyxUtils.subscribeToKey()`'s callback.
      */
     cachedCallbackKey?: OnyxKey;
+
+    /**
+     * The value that triggered the last update
+     */
+    sourceValue?: OnyxValue<OnyxKey>;
```

Comment:
Is it the value or an object of onyx key and value?

Reply:
`OnyxValue` represents a value that can be either a single entry or a collection of entries